tests/test_odcs_requester.py

```diff
@@ -0,0 +1,83 @@
+"""test_odcs_requester.py - test odcs_requester"""
+from typing import Callable
+
+# pylint: disable=redefined-outer-name
+from unittest.mock import MagicMock, create_autospec
+
+import pytest
+from odcs.client.odcs import ODCS, ComposeSourceTag  # type: ignore
+
+from generate_compose.odcs_configurations_generator import (
+    ODCSComposeConfig,
+    ODCSComposesConfigs,
+)
+from generate_compose.odcs_requester import ODCSRequester, ODCSRequestReferences
+
+
+@pytest.fixture()
+def compose_url() -> str:
+    """Example compose-url, as close to the one expected"""
+    return "http://download.eng.bos.redhat.com/odcs/prod/odcs-222222"
+
+
+@pytest.fixture()
+def create_odcs_mock(compose_url: str) -> Callable[[int], MagicMock]:
+    """Create an ODCS mock with specific results for the compose method"""
+
+    def _mock_odcs_compose(num_of_composes: int) -> MagicMock:
+        """Monkey-patched ODCS compose"""
```

Comment:
I don't see any monky patch in here (which is good), so the doc strings should be updated.

Reply:
Will fix, but personally I think we need to disable pylint's demand to include docstrings in`test_*` files. they should be self-explanatory.

Reply:
there is a pytest plugin for that - you can add it in a separate PR and then rebase this...

Reply:
will do